README.md

```diff
@@ -24,6 +24,13 @@ A simple implementation of search-then-sample task and motion planning is provid
 * This repository uses Python versions 3.8+.
 * Run `pip install -r requirements.txt` to install dependencies.
 
+### (Optional) BEHAVIOR Installation
+This repository and some of the functionality it enables are integrated with the [BEHAVIOR benchmark of tasks](https://behavior.stanford.edu/benchmark-guide) simulated with the [iGibson simulator](https://github.com/StanfordVL/iGibson). To install iGibson with BEHAVIOR, follow [these instructions](https://stanfordvl.github.io/behavior/installation.html). Importantly, note that you need to use our forks of the [iGibson](https://github.com/Learning-and-Intelligent-Systems/iGibson) and [bddl](https://github.com/Learning-and-Intelligent-Systems/bddl) repositories. Thus, instead of steps 1a. and 1b. in the linked instructions, run:
+```
+https://github.com/Learning-and-Intelligent-Systems/iGibson.git
```

Comment:
give the full command that people need to run (for instance, i notice there's a `--recursive` flag in the actual installation, do we need that?

Reply:
woops; forgot about this, good catch!